packages/cms-data-sync/src/utils/entries.ts

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-console */
+import { Environment, Entry } from 'contentful-management';
+
+export const clearContentfulEntries = async (
+  contentfulEnvironment: Environment,
+  contentType: string,
+) => {
+  const entries = await contentfulEnvironment.getEntries({
+    content_type: contentType,
+  });
+  console.log(
+    `Cleaning Contentful Entries from content-type ${contentType}...`,
+  );
+
+  await Promise.all(
+    entries.items.map(async (entry) => {
+      if (entry.isPublished()) {
```

Comment:
can we not always unpublish? was just wondering if we need that extra check

Reply:
although it's not flagged in the documentation (https://www.contentful.com/developers/docs/references/content-management-api/#/reference/entries/entry-publishing/unpublish-an-entry/console/js)
<img width="635" alt="Screen Shot 2022-11-23 at 14 46 18" src="https://user-images.githubusercontent.com/16595804/203614919-f977353b-e241-4ab8-8e73-e186e5bde5e3.png">

it gives an error when we try to unpublish a draft entry
<img width="1273" alt="Screen Shot 2022-11-23 at 14 46 29" src="https://user-images.githubusercontent.com/16595804/203615004-2005d422-7722-41bf-a0f6-f2ef36c53141.png">